routes/index.js

```diff
@@ -1,11 +1,12 @@
 const express = require("express");
-
 const router = express.Router();
 
 const userRouter = require("./userRouter");
 const productRouter = require("./productRouter");
+const cartRouter = require("./cartRouter");
 
 router.use("/user", userRouter.router);
 router.use("/products", productRouter.router);
+router.use("/cart", cartRouter.router);
```

Comment:
```suggestion
router.use("/carts", cartRouter.router);
```
다른 브랜치에서 `/products`라고 했던 것 처럼 엔드포인트 컨벤션을 모두 통일시켜주면 좋을 것 같습니다-!